ggml/src/ggml-cpu/llamafile/sgemm.cpp

```diff
@@ -394,8 +393,6 @@ class tinyBLAS {
 
     template <int RM, int RN, int BM>
     NOINLINE void gemm(int64_t m, int64_t n, int64_t BN) {
-        static std::atomic<int64_t> current_chunk;
```

Comment:
I think it should be possible to use the atomic from `params->threadpool->current_chunk`. This is important for performance in systems with non-homogeneous cores.